phpBB/includes/message_parser.php

```diff
@@ -19,6 +19,19 @@
 	exit;
 }
 
+// The following lines are for extensions which include message_parser.php
+// while $phpbb_root_path and $phpEx are out of the script scope
+// which may lead to the 'Undefined variable' and 'failed to open stream' errors
+if (!isset($phpbb_root_path))
```

Comment:
Well, when including from the script where those vars are defined, this would be redundant.